content/learning-paths/cross-platform/eigen-linear-algebra-on-arm/tensorflow-with-sve.md

```diff
@@ -6,54 +6,63 @@ weight: 5
 layout: learningpathall
 ---
 
-## Build Tensorflow with SVE enabled!
+## Build Tensorflow with SVE enabled
 
-Now that you have seen that you can use Eigen with SVE enabled, it's time to build your own SVE-enabled Tensorflow!
+Now that you have seen that you can use Eigen with SVE enabled, it's time to build your own SVE-enabled Tensorflow.
 
-Tensorflow is a very complicated software and even building it requires some effort. However, following the next instructions here should get you up and running with relative ease.
+Tensorflow is a complex application and building it requires significant effort. However, following the instructions below you should be able to build and run it. 
 
 ### Install Build Requirements for Tensorflow
 
-You are going to follow the [Tensorflow Instructions to build from source](https://www.tensorflow.org/install/source), slightly modified.
+You are going to follow the [Tensorflow Instructions to build from source](https://www.tensorflow.org/install/source) with some slight modifications.
 
 Before you attempt to build Tensorflow, you need to install the build dependencies first.
 
-The following packages are needed in a recent Debian/Ubuntu distribution, though you may have to change to the relevant packages if you're using a different distribution:
+The following packages are needed in a recent Debian/Ubuntu distribution. You may have to change to the packages if you're using a different Linux distribution:
 
 ```bash
-# apt -u install clang-17 python3-pip golang python3-virtualenv default-jdk-headless patchelf libhdf5-dev
+sudo apt -u install clang-17 python3-pip golang python3-virtualenv default-jdk-headless patchelf libhdf5-dev -y
```

Comment:
I forgot the clang-17 dependency as that's what I started with, but in the answers I use GCC. The reason is that clang fails at linking due to a bug in ld.gold. This should be changed to `gcc g++` instead.